### ISSUE TITLE
gcc-4.3 and earlier

### ISSUE THREAD

**Issue by rurban** [uxcn/farmhash-c#2]
See my fixes at https://github.com/rurban/smhasher/commit/1a31e8e5e892577363ae575c9ca2094f50f44330

**Comment by uxcn** [comment]
Thanks, I appreciate the heads up.  It looks like I copied a bug by accident.  I'll hopefully push the fix later today.  I'll keep you updated.

**Comment by uxcn** [comment]
I committed fixes for the intrinsics issues.  Specifically, AVX intrinsics were being included for SSSE3 and SSE41 (see https://github.com/uxcn/farmhash-c/commit/595dde89cd5042328146cabf182f3af8fe286df2).  I think this fixes the issue without the explicit version check.

Unrelated, I looked into the collision issue smhasher reported for farmhash32.  The function used to determine acceptable number of collisions is a bit odd (code is [here](https://github.com/rurban/smhasher/blob/master/Stats.h#L191)).  I'm hesitant to change it though.
